src/agnostic/gameState.ts

```diff
@@ -73,6 +84,8 @@ export function advance(gameState: GameState, gameEvent: GameEvent): GameState {
 		delete gameState.players[gameEvent.data];
 	} else if (gameEvent.type === 'inc-player-score') {
 		gameState.players[gameEvent.data.id].score += gameEvent.data.score;
+	} else if (gameEvent.type === 'change-player-name') {
```

Comment:
add an `if` check for the new event type in the `advance` function. the role of the `advance` function is to advance the game state by applying the game event.